crypto/bls12381_utils.c

```diff
@@ -428,17 +498,26 @@ int ep2_read_bin_compact(ep2_t a, const byte *bin, const int len) {
     memcpy(temp, bin, Fp2_BYTES);
     // clear the header bits
     temp[0] &= 0x1F;
-    fp2_read_bin(a->x, temp, Fp2_BYTES);
+    if (fp2_read_bin_safe(a->x, temp, Fp2_BYTES) != RLC_OK) {
```

Comment:
nit: maybe a bit more robust version:
```suggestion
    if (fp2_read_bin_safe(a->x, temp, sizeof(temp)) != RLC_OK) {
```
it does not matter in this version of the code, but in case something ever changes.

Reply:
updated in https://github.com/onflow/flow-go/pull/3558/commits/c0280f8fc204c5ecf3b815c6bb25d3cccf510eb7